### ISSUE TITLE
VictoriaLogs: vmui table resets its settings on request execution

### ISSUE THREAD

**Issue by yuriydzobak** [VictoriaMetrics/VictoriaMetrics#7016]
### Describe the bug

Hi,
Thanks for the new great product, VictriaLogs. I can't believe how low resource it uses and fast.

The Vlogs UI resets table options when you execute it. This creates a bad user experience, especially when you write complex queries and format tables. If you need to retrieve the latest data, the table format resets. It would be great if it could save the selected options in the table columns.

### To Reproduce

1. Open https://play-vmlogs.victoriametrics.com/select/vmui/
2. chnage table setting
3. re-run request again

https://github.com/user-attachments/assets/c4df132e-0c2c-4a3e-a1d6-2c3b128cfb66

### Version

```
~ $ ./victoria-logs-prod  --version
victoria-logs-20240908-191444-tags-v0.29.0-victorialogs-0-gfb339d7574
```

**Comment by hagen1778** [comment]
@Loori-R do you think it is possible to fix?

**Comment by Loori-R** [comment]
Yes, I'll work on fixing this issue.

**Comment by valyala** [comment]
This bug has been fixed in [v0.31.0-victorialogs](https://docs.victoriametrics.com/victorialogs/changelog/). Closing the issue as fixed then.
